flink-contrib/docker-flink/Dockerfile

```diff
@@ -22,9 +22,9 @@ FROM java:8-jre-alpine
 RUN apk add --no-cache bash snappy
 
 # Configure Flink version
-ENV FLINK_VERSION=1.1.1
-ENV HADOOP_VERSION=27
-ENV SCALA_VERSION=2.11
+ARG FLINK_VERSION=1.1.3
```

Comment:
The `README` says I can specify a `FLINK_VERSION` using `docker build --build-arg FLINK_VERSION=1.0.3 flink`, but that is only possible marking the variable as `ARG` and not `ENV`, see https://docs.docker.com/engine/reference/builder/#/arg